modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -68,13 +68,25 @@ def allow_processing?
     end
 
     def va_gov_errored_claims
-      va_gov = ClaimsApi::AutoEstablishedClaim.select(:id, :transaction_id)
-                                              .where(created_at: @search_from..@search_to,
-                                                     status: 'errored', cid: '0oagdm49ygCSJTp8X297')
-                                              .group(
-                                                :id, :transaction_id
-                                              )
-      va_gov.group_by(&:transaction_id)
+      errored_claims = get_unique_errors
+      errored_claims&.group_by(&:transaction_id)
+    end
+
+    def get_unique_errors
+      starting = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+
+      two_days = ClaimsApi::AutoEstablishedClaim.where(created_at: 48.hours.ago..@search_to,
```

Comment:
I think we may be mixing concepts here. The original thought was to query the last 24 hours as that's the retry window VA.gov uses, once we do that, compare the past 30 min with the 23 1/2 hours prior to and compare the uniques, so same concept, but we have a 48 hour window defined here. 

I'm assuming the concept of the "2 days" may be a mixup of us discussing the possibility of trying to use the nightly job, which wouldn't work either way. This is an assumption as to why the 48 hour window is above, as the ticket defined 24 hours